etc/picongpu/pizdaint-cscs/picongpu.profile.example

```diff
@@ -25,19 +25,29 @@ else
     module load PrgEnv-gnu/6.0.4
```

Comment:
This causes a lot of errors after a module purge:
```bash
#right after login:
$ module purge
$ module list
No Modulefiles Currently Loaded.
$ CRAYENV_FOUND=$(module li 2>&1 | grep "PrgEnv-cray" > /dev/null && { echo 0; } || { echo 1; })
$ echo $CRAYENV_FOUND 
1
# therefore go to second case:
$ module load PrgEnv-gnu/6.0.4
PrgEnv-gnu/6.0.4(106):ERROR:105: Unable to locate a modulefile for 'craype-network-aries'
PrgEnv-gnu/6.0.4(228):ERROR:105: Unable to locate a modulefile for 'udreg'
PrgEnv-gnu/6.0.4(228):ERROR:105: Unable to locate a modulefile for 'ugni'
PrgEnv-gnu/6.0.4(39):ERROR:105: Unable to locate a modulefile for 'dmapp'
PrgEnv-gnu/6.0.4(39):ERROR:105: Unable to locate a modulefile for 'gni-headers'
PrgEnv-gnu/6.0.4(39):ERROR:105: Unable to locate a modulefile for 'xpmem'
PrgEnv-gnu/6.0.4(39):ERROR:105: Unable to locate a modulefile for 'job'
PrgEnv-gnu/6.0.4(39):ERROR:105: Unable to locate a modulefile for 'dvs'
PrgEnv-gnu/6.0.4(39):ERROR:105: Unable to locate a modulefile for 'alps'
PrgEnv-gnu/6.0.4(39):ERROR:105: Unable to locate a modulefile for 'rca'
perftools-base/6.5.1(141):ERROR:102: Tcl command execution failed: if { [is_avail rca] } {
    if {[module-info mode load]} {
        module load rca
    }
}
$ module list
Currently Loaded Modulefiles:
  1) gcc/5.3.0             3) cray-libsci/17.06.1   5) atp/2.1.1
  2) craype/2.5.12         4) pmi/5.0.12            6) PrgEnv-gnu/6.0.4

```

Reply:
I am note sure if a `module purge` should be done on such as system *at all*